projects/admin/src/app/circulation/item/item.component.html

```diff
@@ -38,7 +38,7 @@
         <li *ngSwitchCase="'on_loan'">{{ item.status | translate }} <i class="fa fa-arrow-right" aria-hidden="true"></i>
           {{ item.loan.dueDate | dateTranslate :'shortDate' }}</li>
         <li *ngSwitchCase="'in_transit'">{{ item.status | translate }} (to
-          {{ getTransitLocationPid(item) | getRecord: 'locations' : 'field' : 'code' | async }})</li>
+          {{ item.loan.pickup_location.library_name }})</li>
```

Comment:
Not convinced that `pickup_location.library_name` could be used for all cases.
Ok is loan.state == ITEM_IN_TRANSIT_FOR_PICKUP, but is "circulation specialists" can confirm that's OK if loan.state == ITEM_IN_TRANSIT_TO_HOUSE ?

Reply:
You're right, I got piece of info about circulation specialists: 

* if loan is **ITEM_IN_TRANSIT_FOR_PICKUP**, we use *loan pickup_location*
* if loan is **ITEM_IN_TRANSIT_TO_HOUSE**, we use *loan location*

I will adapt this behaviour if possible.